numpy/_import_tools.py

```diff
@@ -1,351 +0,0 @@
-                filedescriptor = ('.pyc', 'rb', 2)
-            else:
-                filedescriptor = ('.py', 'U', 1)
-
-            try:
-                info_module = npy_load_module(fullname + '.info',
-                                              info_file,
-                                              filedescriptor)
-            except Exception as msg:
-                self.error(msg)
-                info_module = None
-
-            if info_module is None or getattr(info_module, 'ignore', False):
-                info_modules.pop(package_name, None)
-            else:
-                self._init_info_modules(getattr(info_module, 'depends', []))
-                info_modules[package_name] = info_module
-
-        return
-
-    def _get_sorted_names(self):
-        """ Return package names sorted in the order as they should be
-        imported due to dependence relations between packages.
-        """
-
-        depend_dict = {}
-        for name, info_module in self.info_modules.items():
-            depend_dict[name] = getattr(info_module, 'depends', [])
-        package_names = []
-
-        for name in list(depend_dict.keys()):
-            if not depend_dict[name]:
-                package_names.append(name)
-                del depend_dict[name]
-
-        while depend_dict:
-            for name, lst in list(depend_dict.items()):
-                new_lst = [n for n in lst if n in depend_dict]
-                if not new_lst:
-                    package_names.append(name)
-                    del depend_dict[name]
-                else:
-                    depend_dict[name] = new_lst
-
-        return package_names
-
-    def __call__(self,*packages, **options):
-        """Load one or more packages into parent package top-level namespace.
-
-       This function is intended to shorten the need to import many
-       subpackages, say of scipy, constantly with statements such as
-
-         import scipy.linalg, scipy.fftpack, scipy.etc...
-
-       Instead, you can say:
-
-         import scipy
-         scipy.pkgload('linalg','fftpack',...)
-
-       or
-
-         scipy.pkgload()
-
-       to load all of them in one call.
-
-       If a name which doesn't exist in scipy's namespace is
-       given, a warning is shown.
-
-       Parameters
-       ----------
-        *packages : arg-tuple
-             the names (one or more strings) of all the modules one
-             wishes to load into the top-level namespace.
-        verbose= : integer
-             verbosity level [default: -1].
-             verbose=-1 will suspend also warnings.
-        force= : bool
-             when True, force reloading loaded packages [default: False].
-        postpone= : bool
-             when True, don't load packages [default: False]
-
-        """
-        # 2014-10-29, 1.10
-        warnings.warn('pkgload and PackageLoader are obsolete '
-                'and will be removed in a future version of numpy',
```

Comment:
Didn't even spot this. 3ish years seems like plenty of time to wait, especially since it seems we broke it anyway